data/scripts/globalevents/server_save.lua

```diff
@@ -42,5 +42,6 @@ function serversave.onTime(interval)
 	addEvent(ServerSaveWarning, 60000, remaningTime)	-- Schedule next event in 1 minute(60000)
 	return not configManager.getBoolean(configKeys.SERVER_SAVE_SHUTDOWN)
 end
-serversave:time("21:50:00")
+
+serversave:time(configManager.getString(configKeys.SERVER_SAVE_TIME))
```

Comment:
I think we can make some checks here, such as if the tag exists in config.lua (just to make sure someone doesn't forget to pull the tag to config, thus generating a warning). Check if it is nil or if it is valid and also generate a warning, and only if none of the conditions are true, return the value of config.

Maybe we can improve a bit more to extract if it is in the correct format of "0:0:0"

```suggestion
local TIME = configManager.getString(configKeys.SERVER_SAVE_TIME)
if TIME == nil or TIME <= 0 then
    Spdlog.error("[serversave.onTime] - serverSaveTime is nil or wrong")
end

serversave:time(TIME)

```